qiskit_ibm_transpiler/wrappers/ai_synthesis.py

```diff
@@ -66,8 +67,8 @@ def transpile(
 
         results = []
         for transpile_resp in transpile_resps:
-            if transpile_resp.get("success") and transpile_resp.get("qasm") is not None:
-                results.append(QuantumCircuit.from_qasm_str(transpile_resp.get("qasm")))
+            if transpile_resp.get("success") and transpile_resp.get("qpy") is not None:
```

Comment:
If we can send the circuits as qasm because of the fallback, couldn’t we receive the result in qasm format too?

if so, we should add it to this part of the method

Reply:
Good catch! I simplified the logic using `deserialize_circuit_from_qpy_or_qasm()` function

Reply:
Excellent! Thanks